CSharpHTTPClient/Client.cs

```diff
@@ -278,9 +280,13 @@ public override bool TryInvokeMember(InvokeMemberBinder binder, object[] args, o
                     {
                         AddRequestHeader((Dictionary<string, string>)obj);
                     }
+                    else if (name == "cancellationToken")
+                    {
+                        cancellationToken = (CancellationToken)obj;
+                    }
                     i++;
                 }
-                result = RequestAsync(binder.Name.ToUpper(), requestBody: requestBody, queryParams: queryParams).ConfigureAwait(false);
+                result = RequestAsync(binder.Name.ToUpper(), cancellationToken, requestBody: requestBody, queryParams: queryParams).ConfigureAwait(false);
```

Comment:
Please move the `cancellationToken` parameter to the end for consistency with our v9 SDK.